content/bump-seed-canonicalization.md

```diff
@@ -148,7 +148,7 @@ pub struct Data {
 
 If you aren't initializing an account, you can still validate PDAs with the `seeds` and `bump` constraints. This simply rederives the PDA and compares the derived address with the address of the account passed in.
 
-In this scenario, Anchor *does* allow you to specify the bump to use to derive the PDA with `bump = <some_bump>`. The intent here is not for you to use arbitrary bumps, but rather to let you optimize your program. The iterative nature of `find_program_address` makes it expensive, so best practice is to store the canonical bump in the PDA account's data upon initializing a PDA, allowing you to reference the bump stored when validating the PDA in subsequent instructions.
+In this scenario, Anchor *does* allow you to specify the bump to use to derive the PDA with `bump = <some_bump>`. The intent here is not for you to use arbitrary bumps, but rather to let you optimize your program. The iterative nature of `find_program_address` makes it expensive, so best practice is to store the canonical bump in the PDA accounts data upon initializing a PDA, allowing you to reference the bump stored when validating the PDA in subsequent instructions.
```

Comment:
We appear to have difference of opinion about using the possessive for accounts. IMO this and the next couple changes should remain "account's" with an apostrophe to indicate belonging to the account. Would you mind letting me know why you feel the opposite?